### ISSUE TITLE
In LDLf, atomic symbols cannot be transformed to automaton.

### ISSUE THREAD

**Issue by cipollone** [whitemech/flloat#25]
### Subject of the issue
Atomic symbols in LDLf don't fully respect the interface `LDLfFormula`. In fact, conversion fails, due to the missing  `_delta()` function.

I think i can quickly address this before merging #24 . 
I'll try to check and use `LDLfPropositionalAtom` instead of `FiniteTraceWrapper`.

### Your environment
- Python version: 3.7.3
- Package Version: commit 91549be (release candidate 0.3.0)

### Steps to reproduce
```python
from flloat.parser.ldlf import LDLfParser

f = LDLfParser()("A & B")
a = f.to_automaton()
```

### Expected behaviour
Conversion to the correct automaton.

### Actual behaviour
TypeError: PLAtomic has ho _delta function.

**Comment by marcofavorito** [comment]
Yes, we can wait for the release, no problem. 
Could we also add a test to cover that case?

**Comment by cipollone** [comment]
> Could we also add a test to cover that case?

Sure! Few tests, just to see that conversion does what is expected an it raises no exception.
Commit 835578c

**Comment by marfvr** [comment]
Fixed by #27
